documentation/docs/20-core-concepts/50-state-management.md

```diff
@@ -117,7 +117,9 @@ Updating the context-based store value in deeper-level pages or components will
 
 If you're not using SSR (and can guarantee that you won't need to use SSR in future) then you can safely keep state in a shared module, without using the context API.
 
-## Component state is preserved
+## Component and Page states are preserved
+
+> In SvelteKit, pages and layouts are components and follow the same rules and limitations as normal Svelte components.
 
 When you navigate around your application, SvelteKit reuses existing layout and page components. For example, if you have a route like this...
```

Comment:
I just realised these two blocks are basically saying the same thing, we should consolidate them

Reply:
Moved some wording around making it clear that layout, page and any components on the page are being reused (this also fits with the code a bit lower where we use `#key` to force a specific component to rerender).

Also included a quick note about `onMount` and `onDestroy`, nor sure we need a code sample for those though.